### PR TITLE
Ds 277 add loggings for scraping error

### PR DESCRIPTION
**Pull request by metanouvelle**
1. add http, dns, and timeout error into log
2. add a status decode column with reason of error

### CODE REVIEW

File: mondu_website_scrapper/settings.py [hunk 1]
```diff
@@ -93,11 +93,12 @@
 
 # start urls input
 START_URLS = [
```
Comment (jmunar) [inline mondu_website_scrapper/settings.py:95]: I see that `START_URLS` is changing with every PR. Why do we need it? I can imagine that URL will always be a dynamic parameter, not a setting

File: mondu_website_scrapper/spiders/catch_fish_scraper.py [hunk 2]
```diff
@@ -318,13 +336,53 @@ def extract_contact_information(
...
+                "scraper_urls": scraper_urls,
                 "contact_information_url": response.url,
                 "phone": list(set(phone)),
                 "email": list(set(email)),
             }
         )
 
+    def errback_httpbin(self, failure):
+        """_summary_
+
```
Comment (jmunar) [inline mondu_website_scrapper/spiders/catch_fish_scraper.py:348]: Please update the docstring

**Review by jmunar — COMMENTED** [review]
Left a couple of minor comments, please review them before approval

File: mondu_website_scrapper/settings.py (hunk 1 above)
Comment (metanouvelle) [reply mondu_website_scrapper/settings.py:95]: The urls in this list are just for running test on my local.

File: mondu_website_scrapper/spiders/catch_fish_scraper.py (hunk 2 above)
Comment (metanouvelle) [reply mondu_website_scrapper/spiders/catch_fish_scraper.py:348]: It comes along with the improvement PR.

**Review by jmunar — APPROVED** [review]
Approved, thanks!

**Pull request merged**